README.md

```diff
@@ -121,9 +151,41 @@ import MD, {
   LicenseBadge,
   LicenseFromPkg,
 } from "jsx-readme";
-import { Heading, InlineCode, LineBreak } from "jsx-md";
+import { Heading, LineBreak } from "jsx-md";
 import pkg from "./package.json";
 
+const installInstructions = `
+Add \`jsx-readme\` and \`ts-node\` to your \`devDependencies\`.
+
+\`\`\`
+npm i jsx-readme ts-node -D
+\`\`\`
+
+Add these configs to your \`tsconfig.json\`:
+
+\`\`\`
+{
+  "compilerOptions": {
+    "resolveJsonModule": true,
+    "jsx": "react"
+   }
+}
+\`\`\`
+
+Create a README.MD template (you may copy the example from this repo examples/README.md.tsx and edit to your taste).
+
+Add the following script to your \`package.json\`:
+
+\`\`\`
+{
+  "scripts": {
+    "generate:readme": "ts-node README.md.tsx"
+  }
+}
+\`\`\`
+
+`;
```

Comment:
Please inline and use the `jsx-md` components to render the markdown:

```tsx
Add <InlineCode>jsx-readme</InlineCode> and <InlineCode>ts-node</InlineCode> to your <InlineCode>devDependencies</InlineCode>.
<LineBreak />
<CodeBlock language='sh'>
npm i jsx-readme ts-node -D
</CodeBlock>
Add these configs to your <InlineCode>tsconfig.json</InlineCode>:
<LineBreak />
<CodeBlock language='json'>
{
  "compilerOptions": {
    "resolveJsonModule": true,
    "jsx": "react"
   }
}
</CodeBlock>
Create a README.MD template (you may copy the example from this repo examples/README.md.tsx and edit to your taste).
Add the following script to your <InlineCode>package.json</InlineCode>:
<LineBreak />
<CodeBlock language='json'>
{
  "scripts": {
    "generate:readme": "ts-node README.md.tsx"
  }
}
</CodeBlock>
```

Reply:
CodeBlock isn´t working as expected. Do you have any suggestion?

![imagem](https://user-images.githubusercontent.com/44852805/104127263-0d951180-5359-11eb-8544-d016b1a42d7f.png)

Reply:
It's due to the `{}` which need to be escaped. You could try the following:

```tsx
<CodeBlock language='json'>
  {`{
  "scripts": {
    "generate:readme": "ts-node README.md.tsx"
  }
}`}
</CodeBlock>
```